stable/prometheus-adapter/values.yaml

```diff
@@ -48,6 +48,7 @@ rules:
 #     matches: ""
 #     as: "my_custom_metric"
 #   metricsQuery: sum(<<.Series>>{<<.LabelMatchers>>}) by (<<.GroupBy>>)
+  existing:
```

Comment:
please put a comment before the new `existing` value explaining what it does.  also this is a feature so should be a minor version bump.

Reply:
@paulczar Added comment as requested